les/costtracker.go

```diff
@@ -138,16 +167,14 @@ func (ct *costTracker) stop() {
 
 // makeCostList returns upper cost estimates based on the hardcoded cost estimate
 // tables and the optionally specified incoming/outgoing bandwidth limits
-func (ct *costTracker) makeCostList() RequestCostList {
+func (ct *costTracker) makeCostList(globalFactor float64) RequestCostList {
 	maxCost := func(avgTime, inSize, outSize uint64) uint64 {
```

Comment:
Since we always pass the `ct.globalFactor()` as the `globalFactor` here, we can simplify this function by removing parameter `globalFactor` and use `ct.globalFactor()` instead.

Reply:
During initialization we generate a cost list with an 1.25 safety factor:
`makeCostList(ct.globalFactor() * 1.25)`
This is needed to calculate minBufLimit / minCapacity which can depend on bandwidth constraints in some cases. The safety factor is applied because globalFactor can change during operation and I did not want to make this value variable.
Actually I wanted to add an extra check to makeCostList so that if globalFactor changes a lot during operation we still announce a cost table to every peer that does not make certain requests impossible. This is an improbable case but then we can be certain that the protocol will operate even in weird conditions with less than optimal performance. I will add it now.

Reply:
Oh I'm sorry. I missed the `minCapacity` calculation. 
Yes you are right and I'm also thinking about it that the `GlobalFactor` can change during in runtime but we calculate the `minCapacity` based on the initial `GlobalFactor`. It doesn't make sense if the `MaxCostTable` which server sends is lower than the `RealCost`. I'm not sure whether this will happen, but the `GlobalFactor` is definitely changing all the time.

Reply:
done

Reply:
It's not correct here. The initial `ct.minBufLimit` is 0, which forcibly sets the `baseCost` and `reqCost` as 0.